src/Sylius/Component/Attribute/Model/AttributeValue.php

```diff
@@ -35,6 +37,11 @@ class AttributeValue implements AttributeValueInterface
     /**
      * @var string
      */
+    protected $localeCode;
+
+    /**
+     * @var string
+     */
     private $text;
```

Comment:
Because when we want to get value from AttributeValue object, we are using `getValue()` method